### PR TITLE
Prefix CLI Option + Docs

### PR DESCRIPTION
**Pull request by barthy-koeln**
Hi,

Thank you for this library. I've tried multiple things and this one worked out very well.

I needed to remove the `I` prefix out of personal taste and made the already existing `prefix` option a CLI parameter.
I added it to the README and took the liberty of improving some details, as well as adding default values to the table.

Please don't hesitate to suggest edits or totally discard this PR.

Greetings
Barthy

### CODE REVIEW

**Comment by Alexnortung** [comment]
Thanks! lgtm. I will review it again later and merge it :)

File: README.md [hunk 1]
```diff
@@ -50,13 +53,14 @@ This can be done with the `--out` flag like in the following example.
...
 ## Flags
 
-| **flag**                    | **Description**                                                                    |
-|-----------------------------|------------------------------------------------------------------------------------|
-| -i, --in <dir>              | The src directory for strapi                                                       |
-| -o, --out <dir>             | The output directory to output the types to                                        |
-| --component-prefix <prefix> | A prefix for components                                                            |
-| -D, --delete-old            | CAUTION: This option is equivalent to running rm -rf on the output directory first |
-| --prettier <file>           | The prettier config file to use for formatting typescript interfaces               |
+| **flag**                    | **Description**                                                                      | default   |
```
Comment (Alexnortung) [inline README.md:56]: Can you put the default column in bold as well?

Comment (barthy-koeln) [reply README.md]: Sure. I also made the headers all capitalized for consistency.

**Review by Alexnortung — APPROVED** [review]
Thanks, looks good!

**Pull request merged**